openssl/src/macros.rs

```diff
@@ -214,3 +214,29 @@ macro_rules! from_pem {
             from_pem, $t, $f);
     }
 }
+
+
+macro_rules! foreign_type_and_impl_send_sync {
+    (
+        $(#[$impl_attr:meta])*
+        type CType = $ctype:ty;
+        fn drop = $drop:expr;
+        $(#[$owned_attr:meta])*
+        pub struct $owned:ident;
+        $(#[$borrowed_attr:meta])*
+        pub struct $borrowed:ident;
+    )
+        => {
+            foreign_type! {
+                type CType = $ctype;
+                fn drop = $drop;
+                pub struct $owned;
```

Comment:
You need to forward the attributes into foreign_type.

Reply:
Okay I will look into that, thanks !